server/src/test/java/org/opensearch/index/mapper/TextFieldTypeTests.java

```diff
@@ -167,6 +167,7 @@ public void testFuzzyQuery() {
 
     public void testIndexPrefixes() {
         TextFieldType ft = createFieldType(true);
+        ft.setIndexAnalyzer(Lucene.STANDARD_ANALYZER);
```

Comment:
Sorry just noticed that, why do we need to set index analyzer explicitly? The default is standard afaik, may be we could have 2 tests instead w/o index analyzer set?

Reply:
That's because the `createFieldType()` method gives a very raw TextFieldType without setting the indexAnalyzer:
https://github.com/opensearch-project/OpenSearch/blob/8aed62e4c56914be1ddc802a118e7fa6039d9dc1/server/src/test/java/org/opensearch/index/mapper/TextFieldTypeTests.java#L71, so if we don't set it explicitly, the indexAnalyzer is null, that will cause `NullPointerException` when we call `delegate.name()` or `delegate.analyzer()`.

Reply:
Oh sorry, I see, it is test code